application/backend/src/main/resources/apps/typerefinery/components/structure/page/clientlibs-header/style.css

```diff
@@ -10,4 +10,8 @@ body {
 
 .bg-secondary {
```

Comment:
namespace?

Reply:
This is a global class; bg-secondary is a class used for background colour; the background colour was using some dark grey values, which we have changed to light blue; this is for the whole app.